integration-templates/kustomer/kustomer-conversations.ts

```diff
@@ -0,0 +1,38 @@
+import type { NangoSync, KustomerConversation } from './models';
+
+export default async function fetchData(nango: NangoSync): Promise<void> {
+    let totalRecords = 0;
+
+    try {
+        const endpoint = '/v1/conversations';
+        const config = {
+            paginate: {
+                type: 'link',
+                link_path_in_response_body: 'links.next',
+                limit_name_in_request: 'pageSize',
+                response_path: 'data',
+                limit: 100
+            }
+        };
+        for await (const conversation of nango.paginate({ ...config, endpoint })) {
+            const mappedConversation: KustomerConversation[] = conversation.map(mapConversation) || [];
+
+            const batchSize: number = mappedConversation.length;
+            totalRecords += batchSize;
+            await nango.log(`Saving batch of ${batchSize} conversation(s) (total conversation(s): ${totalRecords})`);
```

Comment:
is this `log` necessary?

Reply:
@TBonnin , no, it isn't necessary. I just removed it.